src/libraries/Microsoft.PowerFX.STJ/RecalcEngineScopeJSON.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Web;
+using Microsoft.PowerFx;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx
+{
+    public class RecalcEngineScopeJSON
+    {
+#pragma warning disable CA1041
+        [Obsolete]
+        public static RecalcEngineScope FromUri(Engine engine, string uri, ParserOptions options = null)
```

Comment:
We may just take the breaking change and delete these so we can avoid the dependency.